test/unittests/test_dd_synthesis_dc.cpp

```diff
@@ -57,41 +57,23 @@ INSTANTIATE_TEST_SUITE_P(TestDDSynth, TestDDSynthDc,
 TEST_P(TestDDSynthDc, GenericDDSynthesisDcTest) {
     EXPECT_TRUE(readPla(tt, fileName));
 
-    const auto& qc          = DDSynthesizer::synthesize(tt);
-    const auto  totalNoBits = qc->getNqubits();
-
-    // generate the complete truth table.
-    extend(tt);
-    augmentWithConstants(tt, totalNoBits, true);
+    const auto& qc = DDSynthesizer::synthesize(tt);
 
     buildTruthTable(*qc, ttqc);
 
-    EXPECT_TRUE(TruthTable::equal(tt, ttqc));
+    EXPECT_TRUE(TruthTable::equal(ttqc, tt));
```

Comment:
just for safety, maybe also add the symmetric condition down below. So that we can be sure that at least for the examples considered here, the `equals` function is symmetric.
i.e., check `equal(tt, ttqc)` and `equal(ttqc, tt)`.
Please also do so down below in the other test.